### DISCUSSION TITLE
List of compatible IR remote controls

### DISCUSSION THREAD

**Discussion by Renber78** [sle118/squeezelite-esp32#306] [Show and tell]
Here is a list of remote controls that have been tested with Squeeze-ESP32.

Notes:
- For the remote to work on LMS, you must manually place the corresponding mapping file (.ir) on your server to  /usr/share/squeezeboxserver/IR/ and activate it in LMS
- You can use telnet or serial logs to obtain the code for your keys. This will allow you to then edit the mapping file as needed.
- SqueezeESP32 has two protocols (RC5 & NEC) that you must configure on your hardware web page (NVS Editor, set_GPIO). It will not automatically switch between these two protocols, it is one or the other.

Brands that are likely to be compatible:
- RC5: (Old devices) Philips, Magnavox, SBR, Salora, Radiola, Sierra, Loewe, Neufunk, Bluesky, Acec, MAG, Vestel, Kiss
- NEC: Apple, Yamaha, Hitachi, LG, Fisher, Nec, Oki, Orion, Telefunken, Toshiba, Strong, Akai, Dvico

Feel free to share your remote control configuration as well.

**Comment by Renber78** [comment]
![image](https://github.com/sle118/squeezelite-esp32/assets/68775667/c9c1bcc3-cb2d-4a2c-bb5c-50b189bf3b53)![image](https://github.com/sle118/squeezelite-esp32/assets/68775667/61233c46-5bd5-4553-bd59-06aca6484bbd)![image](https://github.com/sle118/squeezelite-esp32/assets/68775667/37092dec-7289-4dd6-89c1-a826fd97eda2)



Brand: Philips 
Model: RT-760/787/790/...
Description: Combi TV/VCR (exists in several forms)
Protocol: RC5

[Philips_Combi_TV_VCR.zip](https://github.com/sle118/squeezelite-esp32/files/12673536/Philips_Combi_TV_VCR.zip)

**Comment by Renber78** [comment]
![image](https://github.com/sle118/squeezelite-esp32/assets/68775667/dfdf2754-cb1e-4f19-949e-c6cccb67e70f)

Brand: Kiss
Model: DP-4xx/5xx/1xxx/...
Description: DVD/Divx Player
Protocol: RC5

[Kiss_DVD.zip](https://github.com/sle118/squeezelite-esp32/files/12673601/Kiss_DVD.zip)

**Comment by Renber78** [comment]
![image](https://github.com/sle118/squeezelite-esp32/assets/68775667/d458b296-3fa4-4eec-9e33-296c4defc3ab)

Brand: DVico
Model: TViX HD
Description: Media Streamer
Protocol: NEC

[DVico_TViX.zip](https://github.com/sle118/squeezelite-esp32/files/12674129/DVico_TViX.zip)

**Comment by holefrog** [comment]
I have two Chinese, they have identical codes.
Brand: unknown
Model: unknown
Description: unknown
Protocol: NEC
[Slim_Devices_Remote.zip](https://github.com/sle118/squeezelite-esp32/files/12899822/Slim_Devices_Remote.zip)
![PXL_20231013_190619009](https://github.com/sle118/squeezelite-esp32/assets/17289895/2264960d-6991-4003-8f92-76daa89907b0)

**Comment by holefrog** [comment]
![PXL_20231014_064308833](https://github.com/sle118/squeezelite-esp32/assets/17289895/58a4f9e7-7d49-4db4-93c6-5c5c6c5206c1)
Brand: Huayu / LG
Model: RM-L1162/AKB74475401
Description: TV Remote
Protocol: NEC
[Huayu-RM-L1162.zip](https://github.com/sle118/squeezelite-esp32/files/12903107/Huayu-RM-L1162.zip)

**Comment by Androssboy** [comment]
Hello!
Please tell me how to link your remote control, which is not in the list of compatible ones? What is the algorithm of actions? What should I use from the code received from the serial port?
![IR](https://github.com/sle118/squeezelite-esp32/assets/103333832/21ac1673-8a98-40cc-b747-3d2a6a65212d)

This is from a different remote
![IR1](https://github.com/sle118/squeezelite-esp32/assets/103333832/b0e815c0-d0ba-4f9f-8cd0-fefd72708180)

**Reply by Renber78** [reply]
Your remote's protocol is simply not supported. (Only NEC and RC5 work).

PS: Please open a new post for your questions because here is not the appropriate place for that

**Comment by Renber78** [comment]
![74fb7185d631240d4068e3accabcf203](https://github.com/user-attachments/assets/d478976c-2111-4e10-b4fe-2dc25bbdd0f8)

Marque : Apple
Modèle : A1294
Description : Apple TV Remote
Protocole : NEC
Note: Limited use


[Apple TV Remote (A1294).zip](https://github.com/user-attachments/files/24466695/Apple.TV.Remote.A1294.zip)
